### DISCUSSION TITLE
Enable compression for Spline

### DISCUSSION THREAD

**Discussion by MelanieCroce** [splinetool/react-spline#100] [Q&A] [answered]
I launched a page speed test of my page containing my animation and I have an error on [pagespeed](https://pagespeed.web.dev/) on the spline link.

I'm NextJs and Gzip compression is done on the rest of my site.

I have no idea how to fix this.

<img width="958" alt="Capture d’écran 2022-12-08 à 15 33 26" src="https://user-images.githubusercontent.com/14973984/206473510-ed3ffe5e-afb8-4d60-9c9a-068cfbdbc0db.png">

My animation : 
`<Spline scene="https://prod.spline.design/ZIJ25XNMlnTavY42/scene.splinecode" />`

    "@splinetool/react-spline": "^2.2.3",
    "@splinetool/runtime": "^0.9.144",

    "next": "13.0.6",
    "react": "18.2.0",
    "react-dom": "18.2.0"

**Comment by marcofugaro** [comment, marked as answer]
You can try downloading the .splinecode file by visiting the link and hosting it yourself.

We don't compress the .splinecode file ok our end because it's already binary.

**Reply by MelanieCroce** [reply]
Thanks. My problem was not only that, my file was also too big. I reduced my file and I hosted it on my side, everything is ok for google!
